resources/views/forum/topics/show.blade.php

```diff
@@ -74,6 +78,14 @@ class="forum-topic-title__title forum-topic-title__title--edit js-forum-topic-ti
 
             @if (count($toolbarItems) > 0)
                 <div class="forum-topic-toolbar js-forum-topic-title--toggleable">
+                    @if (isset($toolbarItems['browse_logs']))
```

Comment:
I think this is better as part of bottom navigation bar otherwise it's rather difficult to access if you're not at the top of the topic already